stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift

```diff
@@ -0,0 +1,109 @@
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import Darwin
+#else
+import Glibc
+#endif
+import CTensorFlow
+import CTensorFlowUnittest
+import TensorFlow
+import StdlibUnittest
+
+typealias CTFServer = OpaquePointer
+
+@usableFromInline
+func debugLog(_ message: @autoclosure () -> String,
+              file: StaticString = #file,
+              line: UInt = #line) {
+  if _RuntimeConfig.printsDebugLog {
+    print("[\(file):\(line)] \(message())")
+    // This helps dump more log before a crash.
+    fflush(stdout)
+  }
+}
+
+
+private class TestCluster {
+  var num_tasks:Int = 0
```

Comment:
Please use camelCase.

Reply:
Nit: Also the name should be `taskCount` instead of `numTasks`.

Reply:
Done.